dev-libs/wlroots/wlroots-0.1.ebuild

```diff
@@ -0,0 +1,124 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+if [[ ${PV} == 9999 ]] ; then
+	EGIT_REPO_URI="https://github.com/swaywm/wlroots.git"
+	inherit git-r3
+	KEYWORDS=""
+else
+	SRC_URI="https://github.com/swaywm/wlroots/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+inherit fcaps meson
+
+DESCRIPTION="Pluggable, composable, unopinionated modules for building a Wayland compositor"
+HOMEPAGE="https://github.com/swaywm/wlroots"
+LICENSE="MIT"
+SLOT="0"
+IUSE="elogind examples filecaps png rootston systemd x11-backend xcb-icccm xcb-xkb xwayland"
+REQUIRED_USE="systemd? ( !elogind )"
```

Comment:
`REQUIRED_USE="^^ ( elogind systemd )"`

Reply:
Sway can run without logind with suid/CAP_SYS_ADMIN, so `-elogind -systemd` should be a valid choice (using ^^ forces the user to select at least one of them).

Reply:
`??`